### ISSUE TITLE
Better setup documentation

### ISSUE THREAD

**Issue by ericlathrop** [Wizcorp/Ejecta-X#89]
Most people using Ejecta-X are going to be web developers who are not especially knowledgeable about Android development. More detailed step-by-step instructions for both Android Studio and Eclipse are needed. Screenshots of what buttons to push would be another big help.

I was able to do steps 1-3 of the README, but I can't understand step 4. I've used phoboslab/Ejecta before, and you just had to paste your javascript and assets into an App folder and build it. I would be happy to write such documentation myself, but I don't know the steps to make it work.

**Comment by matrixreal** [comment]
i also agree

**Comment by stephan-fischer** [comment]
Yes, i have problems also, and i am only a normal web-developer.
I compiled the project folder with the NDK like the description, and opened the project with Android Studio. And then, any suggestion what i should do? I cannot build. I only can  run the gradle build but no apk output for me :(

**Comment by benji011** [comment]
I'm not sure if this is a specific issue but just incase:
If you ever try to run ndk_build but get the message below:

`Android NDK: Could not find application project directory !    
Android NDK: Please define the NDK_PROJECT_PATH variable to point to it.  ` then you have to export the path. I did mine like this:

`export NDK_PROJECT_PATH=/Users/benji0110/Downloads/android-ndk-r9/samples/hello-jni  `

then running ndk-build returns this:

`➜  android-ndk-r9  ./ndk-build -C /Users/benji0110/Ejecta-X                                           
make: Entering directory `/Users/benji0110/Ejecta-X'
Gdbserver      : [arm-linux-androideabi-4.6] libs/armeabi/gdbserver
Gdbsetup       : libs/armeabi/gdb.setup
Install        : libhello-jni.so => libs/armeabi/libhello-jni.so
make: Leaving directory `/Users/benji0110/Ejecta-X'`

Hope this helps, but again, might just be a specific issue.
